src/server/data_store/data_store.h

```diff
@@ -0,0 +1,15 @@
+#ifndef DATA_STORE_H
+#define DATA_STORE_H
+
+#include <string>
+
+// TODO(joejoevictor): We should probably put this into a namespace
```

Comment:
Yes. How about ::protocached::datastore:: namespace?

For your LRU impl it will be in
::protocached::datastore::lru::

For other impl it will be in other child namespace